tools/common-code/toolmanifest.pas

```diff
@@ -233,6 +238,23 @@   TCastleManifest = class
       that can optionally auto-create the source file. }
     property PluginSource: string read FPluginSource;
 
+    { Debian-specific things }
+    { Subfolder of /usr into which the app will be installed:
+      'games' will mean /usr/games/(projectname)
+      We make sure this directory will be set as a working directory when launched
+      Default: games }
+    property DebianInstallFolder: String read FDebianInstallFolder;
+    { App menu section, see man menufile for more information,
+      Default: Games }
+    property DebianSection: String read FDebianSection;
+    { Freedesktop category of the app,
+      see https://www.freedesktop.org/wiki/Specifications/menu-spec/ for more info
+      Default: Game }
+    property DebianCategories: String read FDebianCategories;
```

Comment:
Calling it `DebianCategories` limits the usage. Call it more generic `FreeDesktopCategories`. This way it will be useful for all cases when we need to make `xxx.desktop` file for the game.

Reply:
Done in 2f1630dfc4257ef5d5879b52875dea04200dfc65